tests/unit-tests/enumerations/dim3/connections.cpp

```diff
@@ -15,20 +15,20 @@
 
 namespace specfem::connections_test {
 
-enum class Axes { X, Y, Z };
+enum class Axes2D { X, Y, Z };
```

Comment:
```suggestion
enum class Axis3D { X, Y, Z };
```
?

Probably `Axis<dim>`

Reply:
if you mean axes to perform 2d rotation around, I think it is misleading and should be RotationAxis